students/Chay_Casso/Lesson06/water-regulation-master/waterregulation/test.py

```diff
@@ -0,0 +1,159 @@
+"""
+Unit tests for the water-regulation module
+"""
+
+import sys
+import unittest
+from unittest.mock import MagicMock
+from pump import Pump
+from sensor import Sensor
+from .controller import Controller
+from .decider import Decider
+sys.path.append('../pump')
+sys.path.append('../sensor')
+
+
+class DeciderTests(unittest.TestCase):
+    """
+    Unit tests for the Decider class
+    """
+
+    def test_off_too_high(self):
+        """
+        scenario: PUMP_OFF, above target water level margin
+        :return: PUMP_OUT
+        """
+        decider = Decider(100, 0.05)
+        pump = Pump('127.0.0.1', 8000)
```

Comment:
For values that remain the same through every test like the two above, put them in a `def setUp():` function so you aren't duplicating.